src/kioskboard.js

```diff
@@ -759,11 +759,19 @@
             if (autoScroll) {
               var scrollBehavior = opt.cssAnimations === true ? 'smooth' : 'auto';
               var scrollDelay = opt.cssAnimations === true && typeof opt.cssAnimationsDuration === 'number' ? opt.cssAnimationsDuration : 0;
-              var userAgent = navigator.userAgent.toLocaleLowerCase('en');
               var scrollTop = theInputOffsetTop - (isPlacementTop ? keyboardHeight : 0);
-              if (userAgent.indexOf('edge') < 0 && userAgent.indexOf('.net4') < 0) {
+              var userAgent = navigator.userAgent.toLocaleLowerCase('en');
+              var isEdgeWebView = userAgent.indexOf('edge') > -1 && userAgent.indexOf('webview') > -1;
+              var isBrowserEdgeLegacy = userAgent.indexOf('edge') > -1;
+              var isBrowserInternetExplorer = userAgent.indexOf('.net4') > -1;
+              var isBrowserEdgeWebView = isBrowserEdgeLegacy && userAgent.indexOf('webview') > -1;
+              if ((!isBrowserEdgeLegacy || isBrowserEdgeWebView) && !isBrowserInternetExplorer) {
                 var scrollTimeout = setTimeout(function () {
+                if (isEdgeWebView) {
```

Comment:
Hi @surexxx ,

`isEdgeWebView` is no more needed for defining as a variable and checking. We can use `isBrowserEdgeWebView` instead.

Thanks.